contracts/diaspore/DebtEngine.sol

```diff
@@ -205,7 +210,81 @@ contract DebtEngine is ERC721Base {
             _tokens: paidToken
         });
     }
-    
+
+    function payBatch(
+        bytes32[] _ids,
+        uint256[] _amounts,
+        address _oracle,
+        bytes8 _currency,
+        bytes _oracleData
+    ) external returns (uint256[]) {
```

Comment:
We could declare the paid array directly on the returns statement, also we should include a paidToken array

Reply:
In this case I prefer to create the array in the method to limit its elements. what do you think?